redisvl/index.py

```diff
@@ -164,6 +166,18 @@ def _get_key(self, record: Dict[str, Any], key_field: str = None) -> str:
                 raise ValueError(f"Key field {key_field} not found in record {record}")
         return f"{self._prefix}:{key}"
 
+    @staticmethod
+    def _process_results(results: List["Result"]) -> List[Dict[str, Any]]:
```

Comment:
I feel like this might be useful to some users. Let's make it a utility?

Reply:
yeah good call